src/Platforms/AbstractPlatform.php

```diff
@@ -993,10 +993,18 @@ public function getLocateExpression($str, $substr, $startPos = false)
     /**
      * Returns the SQL snippet to get the current system date.
      *
+     * @deprecated Generate dates within the application.
+     *
      * @return string
      */
     public function getNowExpression()
     {
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/issues/4753',
```

Comment:
Is it deliberate to rely on the redirection from issue to PR? I see you didn't do that in https://github.com/doctrine/dbal/pull/4751

Reply:
No, it needs to be fixed.